photo/queries.py

```diff
@@ -37,49 +40,49 @@ def pictures(self, picture: int = None) -> List[PictureType]:
 
     @strawberry.field
     def picture_comments(
-        self, id: int = None, user_email: str = None, picture_id: int = None
+        self, id: int = None, user: uuid.UUID = None, picture_id: int = None
     ) -> List[PictureCommentType]:
         if id:
             return PictureComment.objects.filter(id=id)
-        if user_email and picture_id:
-            return PictureComment.objects.filter(user=user_email, picture=picture_id)
-        elif user_email:
-            return PictureComment.objects.filter(user=user_email)
+        if user and picture_id:
```

Comment:
Lets refactor this to required the picture id only

Reply:
Implemented